utils/constants.jsx

```diff
@@ -128,6 +128,12 @@ export const Preferences = {
 
     // A/B test preference value
     AB_TEST_PREFERENCE_VALUE: 'ab_test_preference_value',
+
+    ONBOARDING: 'onboarding',
+};
+
+export const OnboardingPreferences = {
+    USE_CASE: 'use_case',
```

Comment:
nit: Had discussion with @hmhealey would be better if we define constants near component 
https://community.mattermost.com/core/pl/qzjd3wuzf7yw9kygzjbcgz4yrr